source/analysis/analysis.do

```diff
@@ -2,17 +2,23 @@ version 14
 clear all
 set more off
 
+
 program main
-    import delimited "build/data/data.txt", delimiters("|") varnames(1) clear
+    yaml read YAML using constants.yaml
```

Comment:
It's a scope issue. It didn't seem to work when I had the stata yaml calls outside of the main function. In politext, I think we normally called the paths variable in the same place where we had our `library()` calls which is what I defaulted to here. 
```
Log created:    2017-02-06 17:09:18
Log completed:  2017-02-06 17:09:19
 

  ___  ____  ____  ____  ____ (R)
 /__    /   ____/   /   ____/
___/   /   /___/   /   /___/   14.2   Copyright 1985-2015 StataCorp LP
  Statistics/Data Analysis            StataCorp
                                      4905 Lakeway Drive
     MP - Parallel Edition            College Station, Texas 77845 USA
                                      800-STATA-PC        http://www.stata.com
                                      979-696-4600        stata@stata.com
                                      979-696-4601 (fax)

Unlimited-user 4-core Stata network license expires 21 Jul 2017:
       Serial number:  501409012939
         Licensed to:  Economics
                       Stanford University

Notes:
      1.  Stata is running in batch mode.
      2.  Unicode is supported; see help unicode_advice.
      3.  More than 2 billion observations are allowed; see help obs_advice.
      4.  Maximum number of variables is set to 5000; see help set_maxvar.


running /Applications/Stata/profile.do ...

. do "source/analysis/analysis.do" 

. version 14

. clear all

. set more off

. 
. yaml read YAML using constants.yaml

. yaml local build_data=YAML.build.data

. yaml local build_analysis=YAML.build.analysis

. 
. program main
  1. 
.     import delimited "`build_data'/data.txt", delimiters("|") varnames(1) clear
  2.     hist x, discrete width(0.5)
  3.     graph export "`build_analysis'/plot.eps", replace
  4. 
.     sum x
  5.     file open  outfile using "`build_analysis'/table.txt", write replace
  6.     file write outfile "<tab:table>" _n
  7.     file write outfile (r(mean)) _n (r(sd)) _n (r(max)) _n (r(min))
  8.     file close outfile
  9. end

. 
. 
. * EXECUTE
. main
file /data.txt not found
r(601);

end of do-file
r(601);
```